src/components/OnchainProviders.tsx

```diff
@@ -16,7 +16,7 @@ function OnchainProviders({ children }: Props) {
     <WagmiProvider config={wagmiConfig}>
       <QueryClientProvider client={queryClient}>
         <OnchainKitProvider
-          apiKey={NEXT_PUBLIC_ONCHAINKIT_API_KEY}
+          apiKey={NEXT_PRIVATE_ONCHAINKIT_API_KEY}
```

Comment:
If you remove `NEXT_PUBLIC_` I'd expect this to be undefined since this is on the client, can you confirm it works?

Reply:
Actually I think we leave this public, reverting